app/styles/about.sass

```diff
@@ -1,36 +1,488 @@
-#about-view
+@import "app/styles/bootstrap/variables"
+@import "app/styles/mixins"
 
-  @import "bootstrap/variables"
+// TODO: Move flat style into probably several files and Bootstrap variables
 
-  .team-column
-    padding-left: 0
+// Variables
 
-    ul.thumbnails
-      margin-left: 40px
-      padding: 0
+$headline-font: 'Arvo', serif
+$body-font: 'Open Sans', sans-serif
 
-      li
-        list-style-type: none
+$burgandy: #7D0101
+$gold: #F2BE19
+$navy: #0E4C60
+$forest: #20572B
+
+.style-flat
```

Comment:
This didn't need to be copied into about.sass. All sass files are joined into one file, so rules placed in any file affect the whole site.